index.js

```diff
@@ -22,13 +23,14 @@ var config = require('./js/mainjs/config.js')(Path.join(__dirname, 'config.json'
 
 // When Electron loading has finished, start the daemon then the UI
 App.on('ready', function() {
+	// Load mainWindow
+	mainWindow = require('./js/mainjs/initWindow.js')(config);
 	// Load tray icon
-	var iconPath = Path.join(__dirname, 'assets', 'icon.png');
+	// OSX has a maximum tray icon height of 18px.
```

Comment:
Where does 18px come from? http://iconhandbook.co.uk/reference/chart/osx/ says the maximum size is 22px. Also is it possible to use the previous larger icon (or vector image if we have that), but display it at a smaller size?